packages/monorepo-scripts/src/deps_versions.ts

```diff
@@ -17,11 +17,31 @@ interface VersionsByDependency {
     [depName: string]: Versions;
 }
 
-const PACKAGE_JSON_GLOB = '../*/package.json';
+interface ParsedDependencies {
+    ignored: VersionsByDependency;
+    included: VersionsByDependency;
+}
+
+const PACKAGE_JSON_GLOB = '../../*/package.json';
+const config = JSON.parse(fs.readFileSync(path.join(__dirname, '../../../package.json')).toString()).config;
+const ignore: string[] = config.ignoreDependencyVersions.split(' ');
+const ignorePackages: string[] = config.ignoreDependencyVersionsForPackage.split(' ');
```

Comment:
Can we optimize for readability over variable name shortness? `ignore` -> `dependenciesWithIgnoredVersions` and `ignorePackages` -> `packagesWithIgnoredDependecyVersions`. It's a mouth-full, and I understand why you chose to go with shorter names but I think this does improve the readability. Open to alternative naming proposals.